src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/BasePage.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.elements.mercury.pages.discussions;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)